### ISSUE TITLE
tg://search?query=Search

### ISSUE THREAD

**Issue by MyGitHub5** [telegramdesktop/tdesktop#25963]
### Steps to reproduce

This is shortcut link to search

### Expected behaviour

Tap on it should go to global search in Telegram

### Actual behaviour

Nothing...

### Operating system

Windows 11

### Version of Telegram Desktop

Latest

### Installation source

Static binary from official website

**Comment by vonDubenshire** [comment]
[tg://search?query=Search](tg://search?query=Search)

**Comment by MyGitHub5** [comment]
Please Fix it

**Comment by MyGitHub5** [comment]
It's also don't work on IOS
For now it's working only on Android...

It's very useful !!!

**Comment by jerrygreen** [comment]
Sometimes issues are inactive much more than this one, and they're still relevant.

Maybe it's not crucial to fix asap as some other issues might be, but it's still neat to have, especially since it's already present on Android. Why not support other platforms?

**Comment by jerrygreen** [comment]
Nope, the search link still doesn't work – just checked the desktop app. On iOS probably doesn't work still, either. Works on android.

**Comment by jerrygreen** [comment]
I hate bots.

**Comment by MyGitHub5** [comment]
Why does it take so long to fix it?!
It's very useful and helps a lot of users...
Take it seriously!!

**Comment by MyGitHub5** [comment]
Waiting for it :)
How long it's gonna take...
